package.json

```diff
@@ -19,31 +19,39 @@
     "lib/"
   ],
   "scripts": {
-    "lint": "eslint index.js lib/ test/add.js test/write.js && jscs index.js lib/ test/add.js test/write.js",
+    "lint": "eslint .",
     "pretest": "npm run lint",
-    "test": "mocha --async-only",
-    "cover": "istanbul cover _mocha --report lcovonly",
-    "coveralls": "npm run cover && istanbul-coveralls"
+    "test": "nyc mocha --async-only"
   },
   "dependencies": {
     "append-buffer": "^1.0.2",
-    "convert-source-map": "^1.5.0",
-    "graceful-fs": "^4.1.6",
-    "normalize-path": "^2.1.1",
-    "now-and-later": "^2.0.0",
+    "convert-source-map": "^1.8.0",
+    "graceful-fs": "^4.2.10",
+    "normalize-path": "^3.0.0",
```

Comment:
Is there a reason we aren't using `path.normalize` from node?

Reply:
`normalize-path` supports both a win32 path and a unix path, and normalize to a unix path.  This behavior is different from `path.normalize` of node.

Reply:
I'll replace `normalize-path` to `path.posix.normalize` after replacing `\\` to `/` in a path if the path is of a win32.
The way to check is `!fp.includes('/') && fp.includes('\\')` because `/` is not allowed in a win32 path
https://learn.microsoft.com/en-us/windows/win32/fileio/naming-a-file

Reply:
👍